bedrock/settings/base.py

```diff
@@ -227,10 +232,25 @@ def lazy_langs():
 LANGUAGES = lazy(lazy_langs, dict)()
 
 FEED_CACHE = 3900
-DOTLANG_CACHE = 600
+DOTLANG_CACHE = config('L10N_CACHE_TIMEOUT', default='2' if DEBUG else '600', parser=int)
```

Comment:
When pulling the latest from this branch I noticed my macbook's fans we're running like crazy. Increasing the default value here from `2` to `10` seemed to calm things down. Not sure if that's the optimal value, but you may want to test a little here for performance.

Reply:
Works for me. It's a balance for sure. Need long enough to cache multiple calls during a request, but short enough not to span multiple requests. 10 should be fine under normal conditions.